### DISCUSSION TITLE
Start Here: Welcome to Linguist Discussions!

### DISCUSSION THREAD

**Discussion by lildude** [github-linguist/linguist#5158] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community and create a community where everyone can help and learn from everyone else.

Please use Discussions to get help with your repository language analysis (we've got a special category for that), ask questions, share ideas and engage with other community members.

Please be courteous and considerate and remember everyone in the community is here on a voluntary basis, even the GitHub staff.

My intention is for this post to be the starting point for Discussions for the Linguist community and plan to update the content with more useful information as time goes on. Please feel free to suggest information you think would be good to know about Linguist that isn't (or doesn't belong) in the [README.md](https://github.com/github/linguist/blob/master/README.md) and I'll consider adding it to this note.

**Comment by AMR2514** [comment]
Escribe un programa que permita al usuario determinar qué número es mayor de tres
número introducidos al azar por éste.
